sconstruct

```diff
@@ -277,11 +292,12 @@ def expandGlobs(files):
 addon = env.NVDAAddon(addonFile, env.Dir('addon'))
 
 langDirs = [f for f in env.Glob(os.path.join("addon", "locale", "*"))]
-
+moFiles = []
```

Comment:
@derekriemer did you test this PR against an add-on which does not have any existing translated files? Because I believe I tried this change when building PR #29, and something down the line (the dependency function?) couldn't handle an empty list.

Reply:
that accidentally got left in from a previous hack. This pr removes moFile from being a dependency of html generation altogether. As it turns out, moFile is assigned inside a for loop iterating over a collection of directories. I don't see why it makes any sense to have html generation depend on 1, and only 1 mo file, from some random language. The reason I made this change is because the mo files are not storing translations from documentation, and I believe this entire process is in error. The translated md files live in doc/[lang]/readme.md and thus it doesn't make sense to make them regenerate if unrelated po files change, as they only need regenerated if the md files change themselves. If we are going to do it this way, annd there's a good reason to do this, the correct solution is to maintaina hash map of language mapping to the object needed to generate that language, and then look up the appropriate mo file for that language to include as a dependency. It doesn't make sense to make every html file depend on a mo file for the last language we saw in lc_messages/lang] as [lang] has nothing to do with all the other html files.